server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1336,6 +1339,81 @@ public Pair<List<MeasurementPath>, Integer> getMeasurementPathsWithAlias(
     return mtree.getMeasurementPathsWithAlias(pathPattern, limit, offset, isPrefixMatch);
   }
 
+  public void exportSchema(File dir) throws IOException, MetadataException {
+    if (!dir.exists()) {
+      dir.mkdirs();
+    } else {
+      if (!dir.isDirectory()) {
+        throw new IOException(String.format("%s is not a directory.", dir.getAbsolutePath()));
+      }
+    }
+    Files.deleteIfExists(
+        FileSystems.getDefault()
+            .getPath(
+                new File(dir.getAbsolutePath(), MetadataConstant.METADATA_LOG).getAbsolutePath()));
+    Files.deleteIfExists(
+        FileSystems.getDefault()
+            .getPath(new File(dir.getAbsolutePath(), MetadataConstant.TAG_LOG).getAbsolutePath()));
```

Comment:
It's better to remind the user there's existing files and let the user delete them.

Reply:
Done. Add test in `testExportSchemaFail`